app/assets/javascripts/backbone/views/log_modal_view.coffee

```diff
@@ -23,7 +23,7 @@ class MS.Views.LogModalView extends Backbone.View
 
   replaceContent: (html) ->
     @$el.html(html)
-    new MS.Views.TranslationsView(el: @$('[data-content-translatable="log"]'))
+    new MS.Views.TranslationsView()
```

Comment:
Not sure if this is a good idea since by hard-coding the scope into TranslationsView, you are (I think) making it impossible to have two translatable field areas on the same page, which seems like it could be a possibility.